LDAR_Sim/src/initialization/leaks.py

```diff
@@ -125,14 +125,23 @@ def generate_initial_leaks(program, site):
         NRd = program['NRd']
         LPR = program['emissions']['LPR']
 
-    n_leaks = random.binomial(NRd, LPR)
+    if program['n_init_leaks'] is not None:
+        n_leaks = program['n_init_leaks']
+    else:
+        n_leaks = random.binomial(NRd, LPR)
+
+    if program['n_init_days'] is not None:
+        pot_active = program['n_init_days']
```

Comment:
[Nit] I'm assuming pot_active stands for potential active here as in like potential days active. I wonder if there is a name we can use to make this more clear. In this case I think using a longer would be fine since we only reference the variable twice. Perhaps something along the lines max days active since that is what the variable is used for?

Let me know your thoughts.

Reply:
Updated.